src/qml/qgismobileapp.qml

```diff
@@ -189,8 +189,8 @@ ApplicationWindow {
             if ( !freehandHandler.active &&
                  point.position.x >= digitizingToolbarCoordinates.x && point.position.x <= digitizingToolbarCoordinates.x + digitizingToolbar.width &&
                  point.position.y >= digitizingToolbarCoordinates.y && point.position.y <= digitizingToolbarCoordinates.y + digitizingToolbar.height ) {
-                // when hovering digitizing toolbar, reset coordinate locator position, nicer UX
-                coordinateLocator.sourceLocation = undefined
+                // when hovering digitizing toolbar, reset coordinate locator position for nicer UX
+                coordinateLocator.sourceLocation = mapCanvas.mapSettings.coordinateToScreen( digitizingToolbar.rubberbandModel.coordinateAt( digitizingToolbar.rubberbandModel.vertexCount - 2 ) );
```

Comment:
Wouldn't a `lastCoordinate` method be nicer to read here than a `vertexCount - 2` (especially the obscure -2 part)? It could even be a readonly property so one could attach things to it

Reply:
Or lastVertexAdded ?